server/data/data-library/src/test/java/org/apache/james/user/lib/UsersRepositoryContract.java

```diff
@@ -53,15 +54,12 @@
 
     class UserRepositoryExtension implements BeforeEachCallback, ParameterResolver {
 
-        private static final boolean ENABLE_VIRTUAL_HOSTING = true;
-        private static final boolean DISABLE_VIRTUAL_HOSTING = !ENABLE_VIRTUAL_HOSTING;
```

Comment:
why removing those explicit variables?

Reply:
It just only uses at 
```java
        public static UserRepositoryExtension withVirtualHost() {
            return new UserRepositoryExtension(ENABLE_VIRTUAL_HOSTING);
        }

        public static UserRepositoryExtension withoutVirtualHosting() {
            return new UserRepositoryExtension(DISABLE_VIRTUAL_HOSTING);
        }
```
And parameter in the constructor is `private final boolean supportVirtualHosting;` , IMO, The name very clear. 
So the `private static` variable is unnecessary. Code will look simpler

Reply:
We tend to use those variables. The thing here is if you take out those variables, then you need to go to the constructor to see what is the boolean is about. While with those, you don't need, you know precisely what they are for.

However, just realize it's for a test contract so I guess ok-ish (but would not be for main code)